src/main/java/uk/gov/hmcts/juror/api/juror/service/JurorServiceImpl.java

```diff
@@ -53,16 +54,17 @@ public class JurorServiceImpl implements JurorService {
     private final JurorDigitalResponseRepositoryMod jurorResponseRepository;
     private final JurorResponseCjsEmploymentRepositoryMod jurorResponseCjsEmploymentRepository;
     private final JurorReasonableAdjustmentRepository jurorReasonableAdjustmentRepository;
-    private final JurorPoolRepository jurorRepository;
+    private final JurorPoolRepository jurorPoolRepository;
     private final UrgencyService urgencyService;
     private final PoolRequestService poolRequestService;
     private final ReasonableAdjustmentsRepository reasonableAdjustmentsRepository;
+    private final JurorRepository jurorRepository;
 
 
     @Override
     public JurorDetailDto getJurorByJurorNumber(final String number) {
         log.debug("Getting juror {} details", number);
-        JurorPool jurorDetails = jurorRepository.findByJurorJurorNumber(number);
+        JurorPool jurorDetails = jurorPoolRepository.findByJurorJurorNumber(number);
```

Comment:
This wont work as the juror could be in multiple pools.

If this is only hit but the bureau officer you can use jurorPoolService.getJurorPoolFromUser(jurorNumber) if it is hit by the public too we will need a chat on this

Reply:
this is just a rename from jurorRepository to jurorPoolRepository